src/sentry/tasks/post_process.py

```diff
@@ -216,10 +216,15 @@ def process_snoozes(group):
     """
     from sentry.models import GroupSnooze, GroupStatus
 
-    try:
-        snooze = GroupSnooze.objects.get_from_cache(group=group)
-    except GroupSnooze.DoesNotExist:
-        return False
+    key = "postprocess_gs:1:%s" % (group.id)
```

Comment:
Unsure if I should use `groupsnooze` as the primary key-space or not since I'm explicitly mixing types in the cache.